mawabot/client.py

```diff
@@ -104,6 +104,12 @@ def run(self):
 
         await self.change_presence(status=discord.Status.invisible)
         logger.info('Setting status to invisible')
+    
+    async def on_resumed(self):
+        ''' Used when the bot reconnects '''
+
+        logger.info('Reconnected - setting status to invisible')
+        await self.change_presence(status=discord.Status.invisible)
```

Comment:
Looks good, but to make both `self.change_presence()`s the same, we should have the `logger.info` call on the same side.

I would personally but the logger call first.